requirements.in

```diff
@@ -0,0 +1,48 @@
+gunicorn
+whitenoise[brotli]
+us  # for US states on profiles
+python-dotenv
+sentry-sdk
+sorl-thumbnail
+    Pillow
+geographiclib  # for maps
+    geopy
```

Comment:
In general I recommend to specify which package was added why

The reasoning is that on few years old projects an upgrade eg from djagno 2 to djagno 3 requires the developer to review each package, find the dead ones, fork them, debug & fix the django 3 compatibility manually - which sometimes takes weeks. That's why we started adding the comments that answer the question "*when can I just drop this package instead of spending days on upgrading it?*"